src/utils/normalization.ts

```diff
@@ -107,3 +107,31 @@ export const normalize = (payload: OperationsRepository) => {
 
   return { apis, documentation } as OperationsRepository;
 };
+
+export const emptyObject = (object: any, preserveValueKeys: string[], ignoreNestedKeys: string[]) => {
+  for (const key in object) {
+    if (typeof object[key] === 'object' && !ignoreNestedKeys.includes(key)) {
+      emptyObject(object[key], preserveValueKeys, ignoreNestedKeys);
```

Comment:
This should be a return right?

Reply:
No, I have a return at the end of the function where it returns the whole object. I keep performing changes within the same object and return the object in the end to the caller so I don't need to return here. returning here would not return me the whole object.